source/Mission.cpp

```diff
@@ -184,12 +185,16 @@ void Mission::Load(const DataNode &node)
 			// This was an "illegal" or "stealth" entry. It has already been
 			// parsed, so nothing more needs to be done here.
 		}
+		else if(child.Token(0) == "cost" && child.Size() >= 2)
```

Comment:
I would suggest renaming `cost` to `player escort daily cost` or `escort daily cost` or smth of the like.
The variable could simply be `dailyCost`

If we do it this way we can just have it be one clear keyword for both things; I don't think we intend to add the possibility of a mission solely with a daily cost here.

That said you can keep the IsPlayerEscort() method, and just have it check if we have a `dailyCost` != 0.